src/components/panel/Account.vue

```diff
@@ -297,6 +303,25 @@
       },
     },
     mounted() {
+      if (this.isAdmin) {
+        if (!this.userIsConnected) {
+          this.$segment.track('ACC View onboarding component - not connected state', {
+            Category: 'Account',
+          });
+        } else if (!this.userIsConnectedAndValidated) {
+          this.$segment.track('ACC View onboarding component - connected not validated state', {
+            Category: 'Account',
+          });
+        } else {
+          this.$segment.track('ACC View onboarding component - connected validated state', {
+            Category: 'Account',
+          });
+        }
+      } else {
+        this.$segment.track('ACC View admin component', {
+          Category: 'Account',
+        });
+      }
```

Comment:
We will have to think about refactoring this part for better readability. Since Segment is eagerly awaited and since it works, it's acceptable

Reply:
you could use a dictionary to simplify the code.

Reply:
sample :
```
    const trackByUserStatus = (status) => {
      const dictionary = {
        userIsNotConnected: () =>  this.$segment.track('ACC View onboarding component - not connected state',{ 
         Category: 'Account',
        });,
        userIsConnectedAndNotValidated: () =>           this.$segment.track('ACC View onboarding component - connected not validated state', {
            Category: 'Account',
          });
        isNotAdminAdmin: () =>         this.$segment.track('ACC View admin component', {
          Category: 'Account',
        });
        userIsConnectedAndValidated: () =>           this.$segment.track('ACC View onboarding component - connected not validated state', {
            Category: 'Account',
          });
      };
      return dictionary[category].call();
    };
```

Reply:
Thanks, I'll do that. :smile: